packages/salesforcedx-apex-replay-debugger/test/unit/commands/apexExecutionOverlayResultCommand.test.ts

```diff
@@ -0,0 +1,786 @@
+/*
```

Comment:
A general note about things in here. There are certain things that are hard coded rather than create a hojillion constants. The interface for the heap dumps is far from trivial and scenarios had to be crafted to ensure things like the SOQL query results and Apex results wouldn't clash (Apex action script tries to commit, this doesn't play well with pending rollbacks from inserts done as part of tests). The code used to generate these and checkpoint locations is at the bottom of the file in comments.